src/CareTogether.Api/OData/LiveODataModelController.cs

```diff
@@ -24,16 +25,22 @@
 
 namespace CareTogether.Api.OData
 {
+    public sealed record Organization([property: Key] Guid Id);
+
     public sealed record Location([property: Key] Guid Id, Guid OrganizationId, string Name);
 
     public sealed record LocationUserAccess(
         [property: Key] Guid UserId,
+        [property: Key] Guid OrganizationId,
         [property: Key] Guid LocationId
     );
 
     public sealed record Family(
         [property: Key] Guid Id,
-        [property: ForeignKey("LocationId")] Location Location,
+        // [property: ForeignKey("OrganizationId")] Organization Organization,
```

Comment:
Why is this commented out?

Reply:
My editor pointed some syntax error and commented out to test another thing before fixing syntax, still uncomment in next commits, thanks for pointing out, this way I will remember